_episodes/03-monitor-the-model.md

```diff
@@ -366,6 +382,17 @@ axes[1].set_ylabel("true sunshine hours")
 {: .language-python}
 ![Scatter plot to evaluate training and test set](../fig/03_regression_training_test_comparison.png)
 
+> ## Exercise: Reflecting on our results
+> 1. Is the performance of the model as you expected (or better/worse)?
+> 2. Is there a noteable difference between training set and test set? And if so, any idea why?
+>
+> > ## Solution
+> >  
+> > While the performance on the train set seems reasonable, the performance on the test set is much worse.
```

Comment:
What hint to the cause of this would you give?

Reply:
We're diving into that in the lesson content that follows of course, but indeed maybe we can give a hint here already.

Reply:
Ok, so this is just an observation.